pipeline-trigger/start_local.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export ADAPTER_API=http://localhost:9000/api/adapter
+export PIPELINE_API=http://localhost:9000/api/pipelines
+
+npm run transpile
+node dist/index.js
```

Comment:
This is usually done by by `.env` files. See ui folder.

Reply:
For the microservices `.env` files are currently not supported, because using `.env` files would require additional tooling, which is provided by the VUE CLI out of the box.

Either the `.env` file must be loaded by the microservice itself or use the [dotenv](https://www.npmjs.com/package/dotenv) package and change the start command in `package.json` to `node -r dotenv/config --enable-source-maps dist/index.js`. 

But I would not make this change in this PR instead, I would do it in another PR for all microservices simultaneously to have a consistent developer experience across all microservices.

Reply:
Thanks for the clarification. Looked like a hack at the first glance.